File: Java/opentraceability/src/main/java/opentraceability/models/events/EventProduct.java
package opentraceability.models.events;

import opentraceability.models.identifiers.EPC;
import opentraceability.utility.Measurement;

public class EventProduct {

    public EPC EPC = null;
    public Measurement Quantity = null;

    public EventProductType Type;

    public EventProduct()
    {

    }

    public EventProduct(EPC epc) {
        this.EPC = epc;
    }
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/models/events/eventproduct.kt
package opentraceability.models.events

import opentraceability.models.identifiers.*
import opentraceability.utility.Measurement

class EventProduct {

    var EPC: EPC? = null
    var Quantity: Measurement? = null
    lateinit var Type: EventProductType

    constructor(){}

    constructor(epc: EPC) {
        this.EPC = epc;
    }

}
